### ISSUE TITLE
WebView browsing and scrolling is a bit jumpy. 

### ISSUE THREAD

**Issue by Maqsood007** [gearvrf/GearVRf-Demos#359]
I have render android WebView in GVR FrameLayout to load my webpages but its experience is jumpy and scrolling is not smooth. i am unable to read it continuously.

`frameLayout = new GVRFrameLayout(this);
frameLayout.setBackgroundColor(Color.TRANSPARENT);
View.inflate(this, R.layout.activity_irisvision_browser, frameLayout);
mWebView = (WebView) frameLayout.findViewById(R.id.webView);
if (mWebView != null) {
mWebView.loadUrl("http://theirisvision.com/media/", preventCaching);
        }` 

And  i inflate the Android webview in GVRFrameLayout.

**Comment by liaxim** [comment]
It must be down to the motion events that get dispatched to the WebView. @rahul27 @parthmehta209 any suggestions, ideas?

**Comment by rahul27** [comment]
@parthmehta209 remember you working on interaction with webviews. Did you notice any issues with the performance?

**Comment by Maqsood007** [comment]
I was unable to render video frames in webview but now i am simply
displaying web pages but performance is slow in respect of clicking links
and scrolling the web pages.

**Comment by rahul27** [comment]
Let me see if I can modify the events sample to test the webview.

**Comment by liaxim** [comment]
@Maqsood007 Do you see any difference with hardware acceleration disabled vs enabled?

**Comment by rahul27** [comment]
@Maqsood007 Could you try the following modified sample and let me know if you see the same issue:

https://github.com/rahul27/GearVRf-Demos/commit/c29636d3d1e7b573ab005288c68f42e67e5100d8

Also could you let me know the GPU on your mobile device? You can use the following adb command to print the GPU info:

`adb shell dumpsys | grep GLES`

**Comment by Maqsood007** [comment]
@rahul27 Yes i tried the GVRWebview instead of Android native Webview now its scrolling and clicking is much batter ..Thanks @rahul27 ..

**Comment by rahul27** [comment]
Awesome. Good to hear. Can we close this issue?

**Comment by Maqsood007** [comment]
Yes we can close it now... but one thing there is lot of difference between
webview experience and samsung internet... why ? Is samsung works on
different architectue... ???

**Comment by thomasflynn** [comment]
Samsung Internet has its own completely custom webview that it uses.  It does not use Android's WebView.

**Comment by Maqsood007** [comment]
Hmmmm but in android webview we are facing a fatal issue in geervr of
rendering videos... do you experience it ?
